modules/custom-status/custom-status.php

```diff
@@ -24,6 +25,8 @@ class Custom_Status extends Module {
 	// This is taxonomy name used to store all our custom statuses
 	const TAXONOMY_KEY = 'post_status';
 
+	const SETTINGS_SLUG = 'vw-custom-status';
```

Comment:
Every module in the new system now has a class-defined `SETTINGS_SLUG` that is used both for the settings page slug and saved module metadata (like `loaded_once`).